ml-app/components/kubeflow-core.jsonnet

```diff
@@ -0,0 +1,60 @@
+local params = std.extVar("__ksonnet/params").components["kubeflow-core"];
```

Comment:
Should we prune the ksonnet app so that it doesn't contain core Kubeflow components?
i.e. should the app assume you already have Kubeflow deployed?